packages/pycharm-community-bin/pycharm-community-bin.pacscript

```diff
@@ -9,9 +9,17 @@ hash="06ef4152d59b97da1186cdf288588d20d69a689fe75b2140130fa5d8326d07ad"
 
 desktop_files_path="$STOWDIR/$name/usr/share/applications"
 application_files_path="$STOWDIR/$name/usr/bin/$pkgname"
+desktop_file_sum="d993cf4b4bce67a1cf7ba7d60e1798eaccbf5381263fbd398d76c261089458d9"
 
 prepare() {
   sudo mkdir -p "$STOWDIR/$name" "$application_files_path" "$desktop_files_path"
+  wget -qO "PyCharm.desktop" "https://raw.githubusercontent.com/ObsidianPresidium/pacstall-programs/master/packages/$name/PyCharm.desktop"
+  if [[ "$(sha256sum PyCharm.desktop | cut -d' ' -f 1)" != $desktop_file_sum ]]
+  then
```

Comment:
Please follow https://github.com/pacstall/pacstall/blob/develop/CONTRIBUTING.md#do-then-and-in-placement=

Reply:
Done!